PatternFileMover/Form1.cs

```diff
@@ -112,12 +112,19 @@ void backgroundWorker1_DoWork(object sender, DoWorkEventArgs eventArgs)
                     {
                         if (!Directory.Exists(data.TargetDirectory + Path.DirectorySeparatorChar))
                         {
-                            // the target directory does not existing
-                            // maybe a broken name association or a network drive is not available
-                            // skip this and go on
+                            string warningMessage = $"Warning: Target directory '{data.TargetDirectory}' does not exist. Possible broken name association.";
+
+                            // Log the warning (if there's a logging system)
+                            Console.WriteLine(warningMessage);
+
+                            // Optionally show a message box in the UI
+                            MessageBox.Show(warningMessage, "Missing Directory", MessageBoxButtons.OK, MessageBoxIcon.Warning);
```

Comment:
I think it would be better if the process is not interrupted and the current behavior of the worker does not change.

What about collecting the broken associations and showing up the message box after finishing the worker?